lib/reducers/adapterReducer.js

```diff
@@ -59,6 +59,13 @@ const ImmutableRoot = Record({
     autoConnUpdate: persistentStore.autoConnUpdate(),
     ignoredDeviceAddresses: Set(),
     errors: List(),
+    showConnectionParams: false,
+    connectionParameters: {
+        minConnectionInterval: 7.5,
+        maxConnectionInterval: 7.5,
+        slaveLatency: 0,
+        connectionSupervisionTimeout: 4000,
```

Comment:
Same 7.5 and 4000 values as mentioned above, so would probably make sense to add constants here somewhere?